storage/inspector/src/main.rs

```diff
@@ -0,0 +1,101 @@
+use libradb::LibraDB;
+use std::path::Path;
+use transaction_builder::get_transaction_name;
+
+// Print out latest information stored in the DB.
+fn print_head(db: &LibraDB) {
+    let version = db
+        .get_latest_version()
+        .expect("get_latest_version throw error");
+
+    println!("Latest Version: {}", version);
+
+    let si = db
+        .get_startup_info()
+        .expect("Can't get startup info")
+        .expect("StartupInfo is empty, database is empty.");
+
+    println!(
+        "The latest ledger info: {}",
+        si.latest_ledger_info.ledger_info()
+    );
+
+    println!("Signatures: {:?}", si.latest_ledger_info.signatures());
+
+    println!(
+        "Current Validator Set: {}",
+        si.latest_validator_set
+            .expect("Unable to determine validator set, DB incorrect")
+    );
+
+    let tx = db
+        .get_transaction_with_proof(version, version, false)
+        .expect("Unable to load latest TXN");
+    println!(
+        "Current Transaction: {}",
+        tx.transaction.format_for_client(get_transaction_name)
+    );
+}
+
+// Some naive implementation to verify local storage consistency by iterating over information
```

Comment:
Maybe you need to give a definition of consistency here.

Reply:
Yeah I am thinking along the lines of:
1. Iterate over all ledgerinfo, transaction info and account tree/eventree to check if we can read that information out.
2. Verify all signatures and hash to make sure all value are not corrupted

More over we can probably check for orphaned nodes exists in rocksdb but not connected , which may signal lost ledgerinfo.